book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -96,7 +96,7 @@ index ac51efd..36c06c8 100755
  1 file changed, 1 insertion(+)
 ----
 
-Jetzt wechseln wir wieder zu unserem Branch `master` und fügen einige Kommentare für die Funktion hinzu.
+Jetzt versuchen wir unserem `whitespace` branch zu mergen, es kommt zu einem Konflikt wegen den Whitespace Änderungen.
```

Comment:
```suggestion
Jetzt versuchen wir unseren `whitespace` branch zu mergen, es kommt zu einem Konflikt wegen der Änderungen an den Whitespaces (dt. Leerzeichen).
```

Reply:
Danke nochmals für das Korrekturlesen!
An diesen Beispelen sieht man mal wieder, dass der Autor (in diesem Fall ich selbst) nicht wirklich in der Lage ist seinen eigenen Text auf Fehler zu überprüfen. Man überliest die Fehler einfach.

Reply:
Das kommt mir bekannt vor 😂. Wen man nur noch das liest was da stehen sollte du nicht mehr das was eigentlich da steht.